src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier extends Figure {
```

Comment:
how about the previous comment? let's remove it
```suggestion
class FigureSupplier {
```